packages/firebase_ml_vision/README.md

```diff
@@ -25,6 +25,19 @@ android {
 }
 ```
 
+### Android
```

Comment:
This label is unnecessary since the last label was `### Android` above.

Reply:
I removed the label